### DISCUSSION TITLE
inpaint fooocus v26 patch not working in main or dev_upstream

### DISCUSSION THREAD

**Discussion by rlr0022** [Panchovix/stable-diffusion-webui-reForge#108] [Q&A]
I cannot seem to get the fooocus inpaint patch to work in reforge. I have reinstalled from scratch and I am using an SDXL checkpoint and the controlnet is set to end @ 0.5. I have looked through discussions and solutions here and on other sites and I cannot seem to get it to work. Any help is appreciated!

The controlnet model is in: stable-diffusion-webui-reForge\models\ControlNet\inpaint_v26.fooocus.patch

Whenever I generate an image I get this wall of text as an error:

This is the beginning:
![Screenshot 2024-08-05 093342](https://github.com/user-attachments/assets/07d875ee-8420-4ede-be58-196b1c3fe2ac)

This is the end:
![Screenshot 2024-08-05 093509](https://github.com/user-attachments/assets/ecbb8c26-8e9f-4f22-9996-2e17e920b45f)

**Comment by rlr0022** [comment]
These are my controlnet settings:

![Screenshot 2024-08-05 094736](https://github.com/user-attachments/assets/6eaad0f3-7014-402e-b880-d181b3f9bd3b)

**Comment by Panchovix** [comment]
Hi there, this may be a fact of a controlnet update I did some days ago.

If you try this commit on main, does it work as expected?

`git checkout e53cf13`

**Reply by rlr0022** [reply]
Thanks for the quick response! Just tried it. It gives the same error as above
